src/main/sensors/gyro.c

```diff
@@ -44,20 +45,29 @@ float gyroADCf[XYZ_AXIS_COUNT];
 static int32_t gyroZero[XYZ_AXIS_COUNT] = { 0, 0, 0 };
 static const gyroConfig_t *gyroConfig;
 static biquadFilter_t gyroFilter[XYZ_AXIS_COUNT];
+static biquadFilter_t gyroFilterNotch[XYZ_AXIS_COUNT];
+static biquadFilter_t gyroFilterOld[XYZ_AXIS_COUNT];
```

Comment:
Yes it's not that user friendly, but not a big problem either.

If you don't hit the center correctly, you will not get the very best result, but still an improvement. The graph above clearly shows a large spectrum of noise removed even around the center.
Spectrum analysis can be easily done in blackbox viewer so you could see it right after a tuning flight and set it for your multirotor, no additional tools needed.

Those who do not want to fiddle around with it can simply leave it disabled and won't see any difference at all. For all with even mediocre noise this could be a large improvement.